x-pack/plugins/security/public/views/overwritten_session/components/overwritten_session_page.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiButton, EuiIcon, EuiSpacer, EuiTitle } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import React, { Component } from 'react';
+
+export interface Props {
+  addBasePath: (path: string) => string;
+  user: User;
+}
+
+interface User {
+  username: string;
+}
+
+export class OverwrittenSessionPage extends Component<Props, {}> {
+  public render() {
+    return (
+      <div className="overwrittenSession">
+        <header className="overwrittenSession__header">
+          <div className="overwrittenSession__content eui-textCenter">
+            <EuiSpacer size="xxl" />
+            <span className="overwrittenSession__logo">
+              <EuiIcon type="logoKibana" size="xxl" />
+            </span>
+            <EuiTitle size="m" className="overwrittenSession__title">
+              <h1>
+                <FormattedMessage
+                  id="xpack.security.overwrittenSession.title"
+                  defaultMessage="You had an active session previously, but logged in as a different user."
```

Comment:
__note:__ wording is the hardest part, would gladly accept any better alternatives.

Reply:
I don't mind the current wording, perhaps @gchaps can provide her input. For some context Gail, we're displaying this screen when the user was previously logged into Kibana as a different user, and they're now logging in as a new user.

Reply:
How about making it a little simpler: 

You previously logged in as a different user.  


**Notes**: 

- In the demo, I noticed this text "Last time you logged in with" in the login screen.  It should be "Last time you logged in **as**"

Reply:
Thanks Gail!

> In the demo, I noticed this text "Last time you logged in with" in the login screen. It should be "Last time you logged in as

Haha, luckily that's Auth0 login page, not Kibana :)